server/src/db/localDB/tables.sql

```diff
@@ -26,7 +26,6 @@ CREATE TABLE "CryptoAccount"
 	ca_id bigserial NOT NULL,
 	ca_number varchar(34) NOT NULL,
 	ca_amount money NOT NULL,
-	ca_last_update time with time zone NOT NULL,
```

Comment:
Why this one is unnecessary? Is it hard to implement or what?

Reply:
Yeah, we don't need this and the second reason I don't have an idea how to create and timestamp in JS and since this attribute is marked as not null, we always have to provide it.